config/locales/searchworks.en.yml

```diff
@@ -42,6 +41,15 @@ en:
         mla: MLA
         turabian: Turabian
         preferred: Preferred citation
+    mini_bento:
```

Comment:
This might just be a me-problem, but it hurts my brain that the ArticleComponent (and these i18n keys) are the thing that renders the catalog, and the catalog renders articles. Is there any hope to make that less confusing?

My only thought is trying to unify things into a single component that's more explicit about what is (or is not) rendering?

```
 render SearchResult::MiniBentoComponent.new(close: true, except: ['articles'])
```

🤷

Reply:
I'm not sure there's a way to make this less confusing.  The mini bento on the articles page, shows catalog and the mini bento on the catalog page shows articles.  It's a confusing pattern.    I'm willing to flip the i18n_keys, but I think the `except: ['articles']` would be even weirder as it's not a set of things, it's a binary choice.